File: 1152/code.c
#include <stdio.h>
#include <string.h>

int main() {

    int num;
    char str[1000];
    int count = 0;

    scanf("%[^\n]", str);

    for(int i=0; i<strlen(str);i++){

        if(str[i] == ' '){

            count++;
            continue;
        }
    }

    printf("%d", count+1);

    return 0;
}

File: 1316/code.py
def word(word) : 

    previous = ''
    alphabet = [0]*26

    

    for i in range(len(word)):
        if word[i] != previous :
            index = ord(word[i]) - ord('a')
            if alphabet[index] ==1 :
        
                return 0
        alphabet[index] = 1
        
        previous = word[i]
    
    return 1

            

num = int(input())
count = 0


for i in range(0, num) :

    word_input = input()
    if word(word_input)==1 :

        count += 1

print(count)




